src/MapReduceResult.php

```diff
@@ -79,7 +79,7 @@ public function getCounts()
      */
     public function getExecutionTimeMS()
     {
-        return (integer) $this->executionTimeMS;
+        return $this->executionTimeMS;
```

Comment:
The constructor will always set `$this->executionTimeMS` to an integer. Should we use an `int` return type on this method and remove the `@return` annotation?

Similarly, `getCounts()` and `getTiming()` can use an `array` return type and omit their `@return` annotations.

Reply:
BC promise prevents us from adding a return type here unfortunately.